src/green/troubleshooting/resetting-the-device-using-sd-card.md

```diff
@@ -25,86 +32,134 @@ Perform this procedure if you want to perform a factory reset on your device.
 
 ## To copy the Home Assistant Green OS to the SD card
 
-1. To download the Home Assistant Green OS installer, paste this [URL](https://github.com/NabuCasa/buildroot-installer/releases/tag/green-installer-20240410) into your browser.
-   - To start the download, select the file *green-installer-20240410.img.xz*.
-   - Download and start [Balena Etcher](https://www.balena.io/etcher).
-      - You may need to run it with administrator privileges on Windows.
-   ![Downloading the required software](/static/img/green/download-circle-outline.svg)
+{% steps %}
+{% step "Downloading the Home Assistant Green OS installer" %}
+{% image "/static/img/green/download-circle-outline.svg" "Downloading the required software" %}
+{% stepContent %}
+
+- To download the Home Assistant Green OS installer, paste this [URL](https://github.com/NabuCasa/buildroot-installer/releases/tag/green-installer-20240410) into your browser.
+  1. To start the download, select the file *green-installer-20240410.img.xz*.
+  2. Download and start [Balena Etcher](https://www.balena.io/etcher).
+     - You may need to run it with administrator privileges on Windows.
+
+{% endstepContent %}
+{% endstep %}
+
+{% step "Selecting the image" %}
+{% image "/static/img/green/etcher1_file.png" "Selecting the image" %}
+{% stepContent %}
 
-2. To select the image, select **Flash from file** and select the image you just downloaded.
+- To select the image, select **Flash from file** and select the image you just downloaded.
```

Comment:
Could this be rephrased?
This 3x "select" makes it strange to read.

Reply:
Thank you for the catch.